static/css/components/carousel.less

```diff
@@ -26,6 +26,11 @@
   overflow-x: scroll;
   padding: 10px 20px;
 
+  a,
```

Comment:
This is _really_ general; it makes me have to check any link on the page in case this is overwriting something, and it makes me check the order of how this CSS file is loaded to see what it might override. Let's avoid that, and put this styling inside the `.carousel__item_blankcover`

Reply:
I think you are looking at the diff and not the full file. The a is nested so this only impacts links in the carousel component.

Reply:
🤦‍♂ You're completely correct! My bad.